core/vm/instructions.go

```diff
@@ -920,6 +920,26 @@ func opSelfdestruct6780(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, erro
 	return nil, errStopToken
 }
 
+func opLog0(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
```

Comment:
Any particular reason to specialize the opLog0?

It's only possible for anonymous event in solidity, which I expect is not frequent used?

Reply:
Specializing t he opLog0 can eliminate code 
https://github.com/ethereum/go-ethereum/blob/master/core/vm/instructions.go#L931
and 
https://github.com/ethereum/go-ethereum/blob/master/core/vm/instructions.go#L934-L937 .
I think.

Reply:
but a loop with 0 iteration doesn't cost too much?